Bio/PDB/Residue.py

```diff
@@ -7,6 +7,8 @@
 
 """Residue class, used by Structure objects."""
 
+from Bio.PDB.Atom import Atom
+from typing import Generator
```

Comment:
While we don't strictly enforce import orders (I think!), it would be nice not to mix standard library imports with internals of Biopython.